samples/helloworld/host/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+add_custom_command(OUTPUT helloworld_u.h helloworld_u.c
+  COMMAND openenclave::oeedger8r --untrusted ${CMAKE_SOURCE_DIR}/helloworld.edl)
+
+add_executable(helloworld_host host.c ${CMAKE_CURRENT_BINARY_DIR}/helloworld_u.c)
+
+target_include_directories(helloworld_host PRIVATE
```

Comment:
Since it is a sample, let's keep being super explicit:

`# Need for the generated file helloworld_u.h`

Reply:
Updated.